### PR TITLE
Add customer and conf api examples

### PR DESCRIPTION
**Pull request by DamianAmbrozkiewicz**

### CODE REVIEW

File: examples/configuration_api_example.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+''' Configuration API client example usage. '''
+
+from livechat.configuration.client import ConfigurationApi
+
+# Get group name.
+configuration_api = ConfigurationApi.get_client(token='<your access token>')
+groups = configuration_api.list_groups()
+group_1 = groups.json()[1]['id']
```
Comment (marcindebski) [inline examples/configuration_api_example.py:8]: In my opinion, `group_1 = groups.json()[1].get('id')` is more Pythonic ;) (applicable to the similar occurrences)

File: examples/configuration_api_example.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+group_1_info = configuration_api.get_group(id=1)
+name = group_1_info.json()['name']
```
Comment (marcindebski) [inline examples/configuration_api_example.py:10]: In my opinion, this example is a little bit too "complex" with "magic" numbers. We should add here something simple, similar to https://developers.livechat.com/docs/management/configuration-api/#list-groups. For example how to get list of existing groups, etc.

File: examples/configuration_api_example.py (hunk 1 above)
Comment (DamianAmbrozkiewicz) [reply examples/configuration_api_example.py:8]: 👍

File: examples/configuration_api_example.py (hunk 2 above)
Comment (DamianAmbrozkiewicz) [reply examples/configuration_api_example.py:10]: 👍 ok, I am going to remove it and keep as following example only: Get list of existing groups.

**Review by marcindebski — APPROVED** [review]

**Pull request merged**